CONTRIBUTING.rst

```diff
@@ -151,8 +151,8 @@ but here is a simple overview:
 
     $ git clone git@github.com:YOUR_GITHUB_USERNAME/pytest.git
     $ cd pytest
-    $ git checkout pytest-2.7      # if you want to fix a bug for the pytest-2.7 series
-    $ git checkout master          # if you want to add a feature bound for the next minor release
+    $ git checkout 2.8.X   # if you want to fix a bug for the pytest-2.8 series
```

Comment:
Perhaps `next-version-features` instead of `future-2.9`? This way we don't have to keep updating the docs as you suggested in the mailing list. (an alternative could be `future-features`, but that reads weird to me :stuck_out_tongue_closed_eyes:).